docs/sources/datasources/tempo.md

```diff
@@ -38,6 +38,12 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
 
 ![Trace to logs settings](/static/img/docs/explore/trace-to-logs-settings-8-2.png 'Screenshot of the trace to logs settings')
 
+### Search
```

Comment:
Should we add docs if it's sill behind a feature flag?

Reply:
Good question, I checked and didn't add docs to any of the other PRs so I'm going to not add them yet